resources/lang/fa.json

```diff
@@ -1,103 +1,154 @@
+    "Media Library": "[*]Media Library",
+    "Media catalog": "کاتالوگ چند رسانه ای",
+    "Messages": "[*]Messages",
+    "My account": "[*]My account",
     "Name": "نام",
     "Name role": "نقش نام",
+    "New password": "[*]New password",
     "No notifications": "بدون اعلان",
+    "Nothing found.": "[*]Nothing found.",
     "Notifications": "اعلانات",
     "Object search": "جستجو اشیا",
     "Oh snap!": "اوه اوه!",
+    "On request: :query": "[*]On request: :query",
+    "Once the account is deleted, all of its resources and data will be permanently deleted. Before deleting your account, please download any data or information that you wish to retain.": "[*]Once the account is deleted, all of its resources and data will be permanently deleted. Before deleting your account, please download any data or information that you wish to retain.",
     "Original referrer: (e.g. google, newsletter)": "ارجاع دهنده اصلی: (به عنوان مثال گوگل ، خبرنامه)",
+    "Overview layouts": "طرح کلی",
     "Paid keywords: (e.g. running+shoes)": "کلمات کلیدی پرداخت شده: (به عنوان مثال دویدن + کفش)",
     "Password": "رمز عبور",
+    "Password changed.": "[*]Password changed.",
+    "Permissions": "[*]Permissions",
     "Please check the entered data, it may be necessary to specify in other languages.": "لطفاً داده های وارد شده را بررسی کنید ، ممکن است لازم باشد به زبان های دیگر مشخص کنید..",
+    "Previously uploaded files": "[*]Previously uploaded files",
     "Product, promo code, or slogan (e.g. spring_sale)": "محصول ، کد تبلیغاتی یا شعار (به عنوان مثال فروش بهاره)",
-    "There are no records in this view": "هیچ رکوردی در این بخش  وجود ندارد",
+    "Profile": "[*]Profile",
+    "Profile Information": "[*]Profile Information",
+    "Profile updated.": "[*]Profile updated.",
+    "Regular user": "[*]Regular user",
     "Remember Me": "مرا به خاطر بسپار",
     "Remove": "حذف",
+    "Remove all": "[*]Remove all",
     "Role": "نقش",
+    "Role display name": "[*]Role display name",
     "Role was removed": "نقش حذف شد",
     "Role was saved": "نقش ذخیره شد",
     "Roles": "نقش ها",
+    "Sample Screen Components": "[*]Sample Screen Components",
     "Save": "ذخیره",
-    "Search": "جستجو",
+    "Scroll to top": "رفتن به بالا",
+    "Search file": "[*]Search file",
     "Search...": "جستجو...",
-    "Send": "ارسال",
-    "Access rights": "حق دسترسی",
+    "Searching results": "[*]Searching results",
+    "See more results.": "[*]See more results.",
     "Sign in to your account": "وارد حسابتان شوید",
+    "Sign in with another user.": "[*]Sign in with another user.",
     "Sign out": "خروج",
-    "Size": "اندازه",
     "Slug": "اسلاگ (Slug)",
-    "Status": "وضعیت",
-    "System name": "نام سیستم",
     "System": "سیستم",
+    "System name": "نام سیستم",
     "System settings": "تنظیمات سیستم",
-    "URL": "آدرس اینترنتی",
+    "Systems": "[*]Systems",
```

Comment:
Hi. Are there any reasons why we should show a `[*]` prefixed string?

Reply:
Next to this the plural "systems" is used, but it should be the singular "system". This was recently changed in any languages.

Reply:
@tabuna this is also still wrong in Russian.

Reply:
There are two steps to produce this PR:
Step 1: scan the source code (src/stubs/views) for `__()` translation function call, which generate the set of translation key string. then look up the original json files for existing translations, if not found, `[*]$key` will be the temporary translation.

Step 2: A machine translation BAIDU is called in to help translate the `temporary translation` from step 1, if the translation is not available from BAIDU, then `temporary translation` will remain, `[*]` indicates  human translation is needed.

As I mentioned, `fa.json` and `uk.json` is not available from BAIDU.